package.json

```diff
@@ -26,7 +26,9 @@
     "commit": "cz",
     "check-secrets": "docker run --rm -v $(pwd):/path zricethezav/gitleaks:v8.17.0 protect --source='/path' --staged --no-banner -v",
     "publish:alpha": "npm run build && npx lerna publish --dist-tag next",
+    "publish:alpha:ignore": "npm run build && npx lerna publish --dist-tag next --ignore-changes '**'",
     "publish:prod": "npm run build && npx lerna publish",
+    "publish:prod:ignore": "npm run build && npx lerna publish --ignore-changes '**'",
```

Comment:
OOC - What are these ignores doing?

Reply:
Is it only to upgrade changed files?

Reply:
This works well:
```
npx lerna publish --dist-tag next --ignore-changes '**' <pkgs>
```

Where pkgs:
```
--force-publish <pkg-name>
```

Updating package.json to have:
- publish:alpha
- publish:alpha:ignore
- publish:prod
- publish:prod:ignore

where `ignore` is the version that ignore changes and allows to release “individual” pkgs.

e.g., deploying API SDK and CW SDK “only”
```
npm run publish:alpha:ignore --force-publish @metriport/api-sdk --force-publish @metriport/commonwell-sdk
```

“only” b/c the dependencies will be released as well, and that makes sense. (edited)